Withdrawal_TestCase_PT_111 Update.

sahajisy opened the pull request:
```
#Author: Sahaj Balgunde
#Keywords Summary : Withdawal Test with Checkbox and Without Checkbox

@PT_111 @WithdrawAccount @Full @Smoke 
#Withdrawal PT_111
Feature: Withdraw user account

Scenario: User is clicking on Top button in Withdrawal Page.

Given Login To Platform Portal
Then Click Withdrawal link on Footer
Then Check Top button on Withdrawal Page

```

Updated 1 test case which was out of scope.

sahajisy committed:
Withdrawal_TestCase_PT_111 Update.

Updated 1 test case which was out of scope.

csonparote commented on platform/src/test/java/platformfeatures/PF_Withdrawal_PT_111.feature:
```diff
@@ -0,0 +1,12 @@
+
+#Author: Sahaj Balgunde
+#Keywords Summary : Withdawal Test with Checkbox and Without Checkbox
+@WithdrawAccount @Full @Smoke @PT_111
```
Putting this tag name @PT_111  here will execute all tags before it .
Plz rearrange tags as per desired execution

sahajisy replied:
Done changes as requested.

csonparote commented on platform/src/test/java/platformfeatures/PF_Withdrawal_PT_111.feature:
```diff
@@ -0,0 +1,12 @@
+#Author: Sahaj Balgunde
+#Keywords Summary : Withdawal Test with Checkbox and Without Checkbox
+@WithdrawAccount @Full @Smoke @PT_111
+#Withdrawal PT_111
```
Remove comments if not needed

sahajisy replied:
Done as requested.

sahajisy committed:
Update PF_Withdrawal_PT_111.feature

csonparote reviewed (approved):
Plz import main into ur branch and test

sahajisy committed:
Merge branch 'main' into Sahaj_Withdrawal_PT_111

sahajisy committed:
Merge branch 'main' into Sahaj_Withdrawal_PT_111

sahajisy committed:
Merge branch 'main' into Sahaj_Withdrawal_PT_111

sahajisy committed:
Deleted Withdrawal_PT_111 Feature File amd Step Definition. Merged Test Cases in Withdrawal.

@csonparote Merged features and Stepdef of PT_111 into Withdrawal feature file and stepdef file.
Deleted Withdrawal_PT111 Feature file and stepdef file to resolve errors.

csonparote reviewed (approved):
Plz import main into your branch and check if working

sahajisy committed:
Merge branch 'main' into Sahaj_Withdrawal_PT_111

sahajisy committed:
Merge branch 'main' into Sahaj_Withdrawal_PT_111

sahajisy committed:
Merge branch 'main' into Sahaj_Withdrawal_PT_111

sahajisy committed:
Merge branch 'main' into Sahaj_Withdrawal_PT_111
